tests/queries/0_stateless/03519_merge_engine_read_column_existing_in_subset_of_tables.sql

```diff
@@ -0,0 +1,84 @@
+drop table if exists test1;
+drop table if exists test2;
+drop table if exists test3;
+drop table if exists test_merge;
+
+create table test1(a UInt64, b UInt64) engine=Memory;
+create table test2(a UInt64, c UInt64) engine=Memory;
+create table test3(a UInt64, d UInt64) engine=Memory;
+create table test_merge (a UInt64, b UInt64, c UInt64, d UInt64, e UInt64) engine=Merge(default, 'test');
```

Comment:
Failed because of random datrabase name in CI
```suggestion
create table test_merge (a UInt64, b UInt64, c UInt64, d UInt64, e UInt64) engine=Merge(database(), 'test');
```